Finals.md

```diff
@@ -0,0 +1,198 @@
+# Finals Schedule Parser
```

Comment:
I think if the docs are in the Wiki, they don't need to be checked in to source control as well. Moreover, having a single canonical version seems preferrable over having two separate copies of the same docs.

Let's remove this file from the PR (assuming the wiki page has the same contents)

Reply:
Accidentally committed this file, it was just meant to be where I could write a draft for the wiki